docs/custom_quantum_functions.md

```diff
@@ -116,29 +116,29 @@ Use the `IS_LED_ON(usb_led, led_name)` and `IS_LED_OFF(usb_led, led_name)` macro
 ```c
 void led_set_user(uint8_t usb_led) {
     if (IS_LED_ON(usb_led, USB_LED_NUM_LOCK)) {
-        PORTB |= (1<<0);
+        writePinLow(B0);
```

Comment:
`|=` sets the bit, so these should all be flipped, no?

Reply:
Yeah, it depends on how the LED is wired.   And if the pin is sit to input/output.